src/ToDo.Droid/Properties/AndroidManifest.xml

```diff
@@ -1,5 +1,33 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android" package="ToDo" android:versionCode="1" android:versionName="1.0">
   <uses-sdk android:minSdkVersion="16" android:targetSdkVersion="30" />
-  <application android:label="ToDo"></application>
+	<uses-permission android:name="android.permission.INTERNET" />
+	<uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+  <application android:label="ToDo">
+	  <activity android:name="microsoft.identity.client.BrowserTabActivity" android:configChanges="orientation|screenSize">
+		  <intent-filter>
+			  <action android:name="android.intent.action.VIEW" />
+			  <category android:name="android.intent.category.DEFAULT" />
+			  <category android:name="android.intent.category.BROWSABLE" />
+			  <data android:scheme="msal980ca8cd-56ec-47dc-972f-a28ccb78e232" android:host="auth" />
+		  </intent-filter>
+	  </activity>
+  </application>
+	<queries>
+		<package android:name="com.azure.authenticator" />
+		<package android:name="ToDo.Droid" />
+		<package android:name="com.microsoft.windowsintune.companyportal" />
```

Comment:
Why is com.azure.authenticator and com.microsoft.windowsintune.companyportal packages required? Are these to support specific auth workflows?